planet/index.html

```diff
@@ -97,7 +104,7 @@
                 </div>
             </div>
         </nav>
-
+        <button id="backToTopBtn" class="light-green lighten-1" onclick="scrollToTop()">Back to Top</button>
```

Comment:
We'll need to either add translation to the string, or maybe use an icon instead?